.github/workflows/test-build.yml

```diff
@@ -1,9 +1,11 @@
 name: Test Build
 
-on:
+on: 
   workflow_dispatch:
-  # pull_request:
-  # push:
+  pull_request:
```

Comment:
do we want to run for ANY pull request or only prs targeting main?

Reply:
I don't see a problem running the test for any PR. Do you have a reason to restrict it to main?

Reply:
Not really tbh; I just know the historic behavior was only build PRs targeting main. But that said, if you don't see an issue with it; neither do I!

Reply:
changed to main only for PR